code/game/objects/structures/displaycase.dm

```diff
@@ -251,3 +290,12 @@
 	start_showpiece_type = /obj/item/clothing/mask/facehugger/lamarr
 	req_access = list(access_rd)
 
+
+/obj/structure/displaycase/trophy
+	name = "trophy display case"
+	desc = "Store your trophies of accomplishment in here, and they will stay forever."
+	obj_integrity = INFINITY // no
+	max_integrity = INFINITY
+	start_showpiece_type = null
+	req_access = list(access_cent_bar)

```

Comment:
Should double check that the centcom official/ert doesn't have this.
Actually, what would even happen if you removed the item? Would it not go back in the pool?